Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +15,72 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {
+            // Splitting the equation into three parts A,B,C
+
+            List<string> tokens = new List<string>();
+            tokens = equation.Split('=','*').ToList();
+            var A = tokens[0];
+            var B = tokens[1];
+            var C = tokens[2];
+
+            //null check on tokens
+            if(A==null || B==null || C==null)
+                return -1;
+
+            // First checking if C contains ? character 
+
+            if(C.Contains("?")){
+                var numeric_A = Convert.ToInt32(A);

```

Comment:
How about operand1 and operand2 instead of numeric_A and numeric_B???
You are trying to parse an equation. Generally in an equation, you have some operands (number may vary based on operation you are performing), an operator which returns you result.

Try to name variables based on problem you are solving.